client/components/chart/index.jsx

```diff
@@ -152,7 +152,7 @@ function Chart( {
 
 	const ChartYAxis = () => (
 		<div ref={ yAxisRef } className="chart__y-axis">
-			<div className="chart__y-axis-width-fix">{ numberFormat( 1e5 ) }</div>
```

Comment:
updated the class name suffix from `fix` -> `spacer` for consistency with the x-axis spacer classname.